onnxruntime_extensions/tools/pre_post_processing/steps/nlp.py

```diff
@@ -0,0 +1,449 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import onnx
+from collections import OrderedDict
+
+from typing import List, Optional, Tuple, Union
+from ..step import Step
+
+
+class SentencePieceTokenizer(Step):
+    def __init__(self, tokenizer, domain: str = "ai.onnx.contrib", name: Optional[str] = None):
+        """
+        Args:
+            tokenizer: SentencePiece tokenizer from huggingface,
+                usually, we can get it by "tokenizer = transformers.AutoTokenizer.from_pretrained(model_name)"
+            domain: Optional domain of step. ["ai.onnx.contrib" or "com.microsoft.extensions"] Defaults to 'ai.onnx.contrib'
+            name: Optional name of step. Defaults to 'SentencePieceTokenizer'
+
+        """
+        super().__init__(
+            ["inputs", "nbest_size", "alpha", "add_bos", "add_eos", "reverse"], ["input_ids", "attention_mask"], name
+        )
+        self.domain = domain
```

Comment:
nit: please follow python naming convention and use '_' prefix for internal values of a class.

Reply:
Done